docs/ADRs/0012-date-time-representation.md

```diff
@@ -0,0 +1,52 @@
+# ADR-0012 Date Time Representation: ISO 8601
+
+## Context
+
+In our software development projects, we have encountered ambiguity related to the representation of dates and times,
+particularly when dealing with time zones. The lack of a standardized approach has led to discussions and possibly
+ambiguity when interpreting timestamps within our applications.
+
+## Assumptions
+
+Standardizing the representation of dates and times will improve clarity and precision in our application's logic and
+user interfaces.
+
+ISO 8601 format is better human-readable than other formats such as unix timestamps.
+
+## Decision
+
+We adopt ISO 8601 with UTC timezone notation (`Z`) as the standard method for representing dates and times in our
+software projects, replacing the usage of Unix timestamps or any other formats or timezones.
+We use both dashes (`-`) and colons (`:`).
+
+We store date and time as: `2024-04-16T16:48:14Z` (always with `Z` as timezone, representing UTC).
```

Comment:
I do understand the need to always use UTC `Z` because it makes life easier. but this means we loose a bit of context around a event. we might want to store information from which timezone an event came in the timestamp.

Reply:
That would be better indeed